persian/otagheh-basteh.md

```diff
@@ -1,2 +1,4 @@
-چشم‌هایم را در یک اتاق کوچک و بسته باز کردم. نگاهی به اطراف انداختم. نمی‌دانستم کجا هستم و چگونه وارد این اتاق شده‌ام.
-هیچ در و پنجره‌ای در این اتاق کوچک ندیدم و هیچ صدایی نمی‌شنیدم. ترسیده بودم و نمی دانستم چکار کنم
\ No newline at end of file
+﻿چشم‌هایم را در یک اتاق کوچک و بسته باز کردم. نگاهی به اطراف انداختم. نمی‌دانستم کجا هستم و چگونه وارد این اتاق شده‌ام.
+هیچ در و پنجره‌ای در این اتاق کوچک ندیدم و هیچ صدایی نمی‌شنیدم. ترسیده بودم و نمی دانستم چکار کنم
+
+بی اختیار بدنم را به شدت به دیوار مقابل کوباندم تا شاید راهی برای رهایی بیابم. اما بجز احساس درد چیزی تغییر نکرد. سر درد شدیدی داشتم.

```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.